iCubTwente01/hardware/motorControl/torso-eb5-j0_2-mc.xml

```diff
@@ -1,90 +1,118 @@
+        <param name="damping">                  0                   0                   0                   </param>
     </group>
+    
+    <group name="CONTROLS">
+        <param name="positionControl">          POS_PID_DEFAULT     POS_PID_DEFAULT     POS_PID_DEFAULT     </param>
+        <param name="velocityControl">          POS_PID_DEFAULT     POS_PID_DEFAULT     POS_PID_DEFAULT     </param>
+        <param name="mixedControl">             POS_PID_DEFAULT     POS_PID_DEFAULT     POS_PID_DEFAULT     </param>
+        <param name="torqueControl">            TRQ_PID_DEFAULT     TRQ_PID_DEFAULT     TRQ_PID_DEFAULT     </param>
+        <param name="currentPid">               2FOC_CUR_CONTROL    2FOC_CUR_CONTROL    2FOC_CUR_CONTROL    </param>
+        <param name="speedPid">                 2FOC_VEL_CONTROL    2FOC_VEL_CONTROL    2FOC_VEL_CONTROL    </param>
+    </group>
+
+
+    <!-- default position PID: begin -->
 
     <group name="POS_PID_DEFAULT">
-        <param name="controlLaw">    Pid_inPos_outPwm                          </param> 
-        <param name="controlUnits">  metric_units               </param> 
-        <param name="pos_kp">            711.11    -1066.66    -1066.66 </param>       
-        <param name="pos_kd">              0.00        0.00        0.00 </param>       
-        <param name="pos_ki">           7111.09   -10666.64   -14222.18 </param>            
-        <param name="pos_maxOutput">       8000        8000       16000 </param>  
-        <param name="pos_maxInt">           750        1000        2000 </param>       
-        <param name="pos_shift">              0           0           0 </param>       
-        <param name="pos_ko">                 0           0           0 </param>       
-        <param name="pos_stictionUp">         0           0           0 </param>       
-        <param name="pos_stictionDwn">        0           0           0 </param>       
-        <param name="pos_kff">                0           0           0 </param>     
+        <param name="controlLaw">             minjerk               </param>
+        <param name="outputType">             pwm                   </param>
+        <param name="fbkControlUnits">        metric_units          </param>
+        <param name="outputControlUnits">     machine_units         </param>
+        <param name="kp">           -711.11     -1066.66    -1066.66    </param>
+        <param name="kd">           0.00        0.00        0.00        </param>
+        <param name="ki">           -7111.09    -10666.64   -14222.18   </param>
+        <param name="maxOutput">    8000        8000        16000       </param>
+        <param name="maxInt">       750         1000        2000        </param>
+        <param name="stictionUp">   0           0           0           </param>
+        <param name="stictionDown"> 0           0           0           </param>
+        <param name="kff">          0           0           0           </param>
```

Comment:
The signs of the pids are changed :warning:

Reply:
fixed